### ISSUE TITLE
Progress bar doesn't properly resize

### ISSUE THREAD

**Issue by gdevenyi** [sahib/rmlint#287]
Running a rmlint process in a tmux and attaching/detaching on different devices, when the are less than some minimum column limit, text is chopped off and lost, rather than shrinking the progress bar further to allow for the text (or adding a newline)

rmlint 2.6.1 on Ubuntu Trusty.

**Comment by gdevenyi** [comment]
Example:

Too narrow
```
▕▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒░░░░░░░░░░░░░▏Matching (138417 dupes of 132233 originals; 2576.14 GB to scan in 4579
```

Wide enough
```                                         
▕▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▏                                        Matching (1338585 dupes of 132400 originals; 2574.20 GB to scan in 4575120 files, ETA: 1d 19h 3m 47s)

```

**Comment by sahib** [comment]
Hello @gdevenyi,

your proposed solution only helps until the progressbar can't shrink anymore. Then the text would be of course truncated again. I guess your statement is: Prefer text over progressbar?
Printing newlines is not an option from my perspective. I doubt it would look good and being able to write to multiple lines at the same time would make the code a lot more complex since we would need to use something like ncurses for that.

I can lower the minimal size of the progressbar further. Is this something that would solve this for you?

**Comment by gdevenyi** [comment]
Yes, I much prefer the text over the progressbar, that's where the information is. I totally understanding wanting to avoid the newline, that indeed can be very complicated.

**Comment by sahib** [comment]
Hello @gdevenyi,

this should be implemented as of 6e1d9944. Now the progressbar begins to shrink when less than 120 chars width are available. At about 60 chars it should vanish (mostly).
